crates/krun/src/bin/krun.rs

```diff
@@ -206,10 +208,44 @@ fn main() -> Result<()> {
         "PATH",                        // needed by `krun-guest` program
         "RUST_LOG",
     ];
+
+    // https://github.com/AsahiLinux/docs/wiki/Devices
+    const ASAHI_SOC_DEV_IDS: [&str; 12] = [
```

Comment:
```suggestion
    const ASAHI_SOC_COMPAT_IDS: [&str; 12] = [
```

My bad. dev_id / device id has a specific meaning, which would be confusing in this case.